app/renderer/js/preload.js

```diff
@@ -13,9 +13,19 @@ require('./notification');
 // Prevent drag and drop event in main process which prevents remote code executaion
 require(__dirname + '/shared/preventdrag.js');
 
-// eslint-disable-next-line camelcase
+// we have and will have some non camelcase stuff
+// while working with zulip and electron bridge
+// so turning the rule off for the whole file.
+/* eslint-disable camelcase */
+
 window.electron_bridge = require('./electron-bridge');
 
+// Indicates if the user is idle or not
+window.electron_bridge.idle_on_system = false;
+
+ // Indicates the time at which user was last active
+window.electron_bridge.last_active_on_system = Date.now();
```

Comment:
Set this instance variables, in the electron_bridge class constructor. You will need to call `super()` in the first line.

Reply:
@priyank-p do you want me to construct a subclass of electron_bridge class and use it here in preload.js ?

Reply:
No, what I am trying to say is initialize the variables in constructor of `electron_bridge` class.

Reply:
ok. got it.